src/App.js

```diff
@@ -7,9 +7,12 @@ import { getConferenceList } from "./api";
 import Navbar from "./Navbar";
 import { BrowserRouter } from "react-router-dom";
 
-const StyledApp = styled.div`
+const StyledApp = styled.body`
   text-align: center;
-  background: lightgrey;
+  background: #ccb7bb;
+  width: auto;
+  height: 1000px;
```

Comment:
out of interest, why have you set the height to 1000px for the body?

Reply:
We have set the height of the background to make sure it covers the whole page and not just the objects within it, is this okay?

Reply:
Have a look at `vh` I prefer using this as it allows you to set the height to the users screen